docs/admin/pool_operations.md

```diff
@@ -1244,6 +1254,29 @@ to restore redundancy on the remaining engines.
     the --force option. Please note that forcing the operation may result in data loss,
     and it is strongly recommended to verify the RF status before proceeding.
 
+#### System Exclude
+
+To exclude ranks or hosts from all pools that they belong to, the 'dmg system exclude'
+command can be used. The command takes either a host-set or rank-set:
+
+To exclude a set of hosts from all pools (excludes all ranks on selected hosts):
+
+```Bash
+$ dmg system exclude --rank-hosts foo-[001-100]
+```
+
+To exclude a set of ranks from all pools:
+
+```Bash
+$ dmg system exclude --ranks 1-100
+```
+
+Selected ranks will be set to `AdminExcluded` state in system membership (as shown in output of
+`dmg system query` command). This will exclude all rank targets from all pools and render the ranks
+unusable in pools until `dmg system clear-exclude --ranks=${rank_range_list}` is run to return the
+ranks to the previous (normally `Excluded`) state. Then the engine can be restarted to rejoin the
+system and will enter the `Joined` state and ready to be reintegrated into pools.
```

Comment:
It could be indicated that the engine will rejoin automatically the system and also indicate how much time it took in general.  Could be helpful to add an example of dmg command usage for checking if the engines have enter the `Joined` state.

Reply:
time is not easy to determine as it depends on the size of the cluster and lots of other factors. I can add the syntax for system query. Done